microsoft/fluentui
# experiment: preact test setup

Looking at testing `@fluentui/react-button` with Preact. Not currently working.

Testing steps:
1. Clone this branch
2. From repo root: `yarn` to install deps
3. `yarn nx run @fluentui/react-button:test` to run tests

It looks like tests are loading the ESM version of Preact but I'd expect them to load the CJS version.

Preact docs on Jest setup: https://preactjs.com/guide/v10/getting-started#aliasing-in-jest
